ui/components/Breadcrumbs.tsx

```diff
@@ -25,21 +26,29 @@ export const Breadcrumbs = () => {
       </Link>
       {parentValue !== currentPage && parsed.name && (
         <>
-          <Icon
-            type={IconType.NavigateNextIcon}
-            size="large"
-            color="neutral40"
-          />
-          <Link
-            to={formatURL(currentPage, parsed)}
-            textProps={{ size: "large", color: "neutral40" }}
-          >
-            {parsed.name}
-          </Link>
+          {label && (
```

Comment:
Due to the navigation (breadcrumb) issue and it's all a static routes and only display the name from the queryParams as it's 
` Parent > child ` and there's no proper way to set the **Breadcrumb route** for this page as ` Application > (hr/kustomization) automation details > violation message ` beside it won't make any sense to display the violation details as `Application > violations` so that we went for displaying only the `name` and as there won't be a parent here then UI show a **>** before the name so that check was needed for better visualization .


I'm working on moving the breadcrumb to Page so that we can pass the routes easily for derived routes like the way we do in WGE

Reply:
Why not just leave the Breadcrumbs as they are? Isn't this just another tab on an automation?

Reply:
Yes because the violations is now a tab in the AutomationDetails for hr and kustomization so the breadcrumb should follow

`Applications > flux-system > Container Running As User in deployment helm-controller (1 occurrences) ` 

this not doable in current OSS as mentioned